src/javascript/app/pages/trade/barriers.js

```diff
@@ -198,6 +198,17 @@ const Barriers = (() => {
         });
     };
 
+    const isAbsoluteZero = num =>  {
+        if (parseFloat(num) !== 0) {
+            return false;
+        } else if (Object.is(parseFloat(num), -0)) {
+            return false;
+        } else if (num.toString().charAt(0) === '+') {
+            return false;
+        }
+        return true;
```

Comment:
so it would be something like:
```
const isAbsoluteZero = num => !(parseFloat(num) !== 0 || /+|-/.test(num.toString().charAt(0)));
```

Reply:
in that case it doesn't need to be a function, it can be a variable and moved to above `const empty`

Reply:
That looks leaner! I'll update it